lib/index.js

```diff
@@ -96,7 +111,9 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       const {actor} = req.user || {};
       const record = await brAccount.get({actor, id});
       res.status(200).json(record);
-    }));
+    }
+    )
+  );
```

Comment:
This is some funky stuff... do we need to fix the eslint rules so it doesn't happen?

Reply:
fixed! will look into it a linter issue for this in a moment.